src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -62,10 +69,11 @@ private FirebaseAuth(FirebaseApp firebaseApp) {
    */
   @VisibleForTesting
   FirebaseAuth(
-      FirebaseApp firebaseApp, GooglePublicKeysManager googlePublicKeysManager, Clock clock) {
+      final FirebaseApp firebaseApp, GooglePublicKeysManager googlePublicKeysManager, Clock clock) {
```

Comment:
I think it's some left over change from a prototyping work I was doing. Removed.